feature-gate/Dockerfile

```diff
@@ -20,10 +24,13 @@ RUN cp $REPO_ROOT/$SERVICE/target/release/feature-gate /opt/feature-gate
 
 # Create minimal deployment
 FROM debian:buster-slim as deployment
-ARG DEBIAN_FRONTEND=noninteractive
+ARG SERVICE
+ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update -q \
     && apt-get install -y -q "libssl1.1" "tini" "libpq-dev" \
     && rm -rf /var/lib/apt/lists/*
 COPY --from=builder /opt/feature-gate /usr/bin/feature-gate
+COPY $SERVICE/config.default.toml /etc/architus/config.toml
 ENV RUST_BACKTRACE=1
 ENTRYPOINT ["/usr/bin/tini", "--", "/usr/bin/feature-gate"]
+CMD ["/etc/architus/config.toml"]
```

Comment:
This lets the config file path get overridden (but not the "entrypoint"); see the production Kubernetes manifest for an example where it gets overridden to a ConfigMap mount point.